tests/test_inotify.py

```diff
@@ -166,7 +166,10 @@ def test__cycle(self):
 
             i = inotify.adapters.InotifyTree(path)
 
-            with open('seen_new_file1', 'w'):
+            watches = i._i._Inotify__watches
```

Comment:
Can you give me an overview on why this file has to change (aside from fully-qualifying the paths)?